libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -330,12 +330,12 @@ void PbftManager::setPbftStep(size_t const pbft_step) {
   if (step_ > MAX_STEPS && LAMBDA_backoff_multiple < 8) {
```

Comment:
we can even remove this whole if (...) else ...

Reply:
fixed, thanks